src/main/java/frc/utils/PIDObject.java

```diff
@@ -0,0 +1,97 @@
+package frc.utils;
+
+
+public class PIDObject {
+
+	private double kp, ki, kd, ff;
+
+	private double dFilter;
+
+	private double iZone;
```

Comment:
what does it mean?

Reply:
changed to errorZoneForIToAccumulate

> The I portion of the PID controller basically increases with error over time and if you do not constrain it to a zone sometimes it can become a large value that causes unwanted instability. The Izone is essentially an error band that limits when the I value starts to accumulate.